.github/workflows/prober-test.yml

```diff
@@ -0,0 +1,37 @@
+name: Prober Unit Test
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+    - 'cmd/prober/**'
+  push:
+    branches:
+      - main
+    paths:
+    - 'cmd/prober/**'
+
+permissions:
+  contents: read
+
+jobs:
+  prober-test:
+    name: 'Prober test'
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
```

Comment:
id-token for iam, but does the prober cli know to use the github oidc identity?

Reply:
yup, prober imports all providers that cosign supports

https://github.com/sigstore/scaffolding/blob/599588eb7a1be9257bb629edb05e778b095cd9ec/cmd/prober/prober.go#L30